src/scanner-engine.ts

```diff
@@ -136,11 +136,16 @@ export function runScannerEngine(
 
   return new Promise<void>((resolve, reject) => {
     child.on('exit', code => {
-      if (code === 0) {
-        log(LogLevel.DEBUG, 'Scanner engine finished successfully');
-        resolve();
+      if (typeof code === 'number') {
```

Comment:
is it possible that code is not a number?

Reply:
Seems not, it is documented as a `Number` type 
https://nodejs.org/api/process.html#event-exit
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#number_type

Reply:
so we don't actually need that conditional, right? Aside of that, LGTM